2022-08-21 19:29:24,309 [ERROR anvil.app-server.run] Error report from server code:
TypeError: list indices must be integers or slices, not str
Traceback:
  C:/Users/markd/PycharmProjects/fantasy-baseball-scramble/mlb_api.py:41

  C:/Users/markd/PycharmProjects/fantasy-baseball-scramble/mlb_api.py:57

  MyCRUDApp/ServerModule1.py:20

  c:/users/markd/appdata/local/programs/python/python38/lib/site-packages/anvil_downlink_worker/anvil/_server.py:1441

2022-08-21 19:30:28,447 [ERROR anvil.app-server.run] Error report from server code:
TypeError: list indices must be integers or slices, not str
Traceback:
  C:/Users/markd/PycharmProjects/fantasy-baseball-scramble/mlb_api.py:41

  C:/Users/markd/PycharmProjects/fantasy-baseball-scramble/mlb_api.py:57

  MyCRUDApp/ServerModule1.py:21

  c:/users/markd/appdata/local/programs/python/python38/lib/site-packages/anvil_downlink_worker/anvil/_server.py:1441

2022-08-21 19:30:55,915 [ERROR anvil.app-server.run] Error report from server code:
AttributeError: 'dict' object has no attribute 'read'
Traceback:
  c:/users/markd/appdata/local/programs/python/python38/lib/json/__init__.py:293

  MyCRUDApp/ServerModule1.py:17

  c:/users/markd/appdata/local/programs/python/python38/lib/site-packages/anvil_downlink_worker/anvil/_server.py:1441

2022-08-21 19:31:57,137 [ERROR anvil.app-server.run] Error report from server code:
AttributeError: 'dict' object has no attribute 'read'
Traceback:
  c:/users/markd/appdata/local/programs/python/python38/lib/json/__init__.py:293

  MyCRUDApp/ServerModule1.py:17

  c:/users/markd/appdata/local/programs/python/python38/lib/site-packages/anvil_downlink_worker/anvil/_server.py:1441

2022-08-21 19:32:38,221 [ERROR anvil.app-server.run] Error report from server code:
AttributeError: 'dict' object has no attribute 'read'
Traceback:
  c:/users/markd/appdata/local/programs/python/python38/lib/json/__init__.py:293

  MyCRUDApp/ServerModule1.py:17

  c:/users/markd/appdata/local/programs/python/python38/lib/site-packages/anvil_downlink_worker/anvil/_server.py:1441

2022-08-21 19:34:54,494 [ERROR anvil.app-server.run] Error report from server code:
AttributeError: 'dict' object has no attribute 'read'
Traceback:
  c:/users/markd/appdata/local/programs/python/python38/lib/json/__init__.py:293

  MyCRUDApp/ServerModule1.py:19

  c:/users/markd/appdata/local/programs/python/python38/lib/site-packages/anvil_downlink_worker/anvil/_server.py:1441

2022-08-21 19:35:10,752 [ERROR anvil.app-server.run] Error report from server code:
AttributeError: 'dict' object has no attribute 'read'
Traceback:
  c:/users/markd/appdata/local/programs/python/python38/lib/json/__init__.py:293

  MyCRUDApp/ServerModule1.py:19

  c:/users/markd/appdata/local/programs/python/python38/lib/site-packages/anvil_downlink_worker/anvil/_server.py:1441

2022-08-21 19:36:13,869 [ERROR anvil.app-server.run] Error report from server code:
TypeError: the JSON object must be str, bytes or bytearray, not dict
Traceback:
  c:/users/markd/appdata/local/programs/python/python38/lib/json/__init__.py:341

  MyCRUDApp/ServerModule1.py:19

  c:/users/markd/appdata/local/programs/python/python38/lib/site-packages/anvil_downlink_worker/anvil/_server.py:1441

2022-08-21 19:36:47,724 [ERROR anvil.app-server.run] Error report from server code:
TypeError: the JSON object must be str, bytes or bytearray, not dict
Traceback:
  c:/users/markd/appdata/local/programs/python/python38/lib/json/__init__.py:341

  MyCRUDApp/ServerModule1.py:19

  c:/users/markd/appdata/local/programs/python/python38/lib/site-packages/anvil_downlink_worker/anvil/_server.py:1441

2022-08-21 19:37:32,064 [ERROR anvil.app-server.run] Error report from server code:
TypeError: list indices must be integers or slices, not str
Traceback:
  C:/Users/markd/PycharmProjects/fantasy-baseball-scramble/mlb_api.py:41

  C:/Users/markd/PycharmProjects/fantasy-baseball-scramble/mlb_api.py:59

  MyCRUDApp/ServerModule1.py:22

  c:/users/markd/appdata/local/programs/python/python38/lib/site-packages/anvil_downlink_worker/anvil/_server.py:1441

2022-08-21 19:38:04,667 [ERROR anvil.app-server.run] Error report from server code:
TypeError: list indices must be integers or slices, not str
Traceback:
  C:/Users/markd/PycharmProjects/fantasy-baseball-scramble/mlb_api.py:41

  C:/Users/markd/PycharmProjects/fantasy-baseball-scramble/mlb_api.py:59

  MyCRUDApp/ServerModule1.py:22

  c:/users/markd/appdata/local/programs/python/python38/lib/site-packages/anvil_downlink_worker/anvil/_server.py:1441

2022-08-21 19:39:43,784 [ERROR anvil.app-server.run] Error report from server code:
TypeError: string indices must be integers
Traceback:
  C:/Users/markd/PycharmProjects/fantasy-baseball-scramble/mlb_api.py:11

  C:/Users/markd/PycharmProjects/fantasy-baseball-scramble/mlb_api.py:61

  MyCRUDApp/ServerModule1.py:22

  c:/users/markd/appdata/local/programs/python/python38/lib/site-packages/anvil_downlink_worker/anvil/_server.py:1441

2022-08-21 19:41:04,258 [ERROR anvil.app-server.run] Error report from server code:
IndexError: list index out of range
Traceback:
  C:/Users/markd/PycharmProjects/fantasy-baseball-scramble/mlb_api.py:61

  MyCRUDApp/ServerModule1.py:22

  c:/users/markd/appdata/local/programs/python/python38/lib/site-packages/anvil_downlink_worker/anvil/_server.py:1441

2022-08-21 19:47:43,077 [ERROR anvil.app-server.run] Error report from server code:
FileNotFoundError: [Errno 2] No such file or directory: 'leagues/league4/team-lineups/Liverpool Ale Quaffers.json'
Traceback:
  MyCRUDApp/ServerModule1.py:26

  c:/users/markd/appdata/local/programs/python/python38/lib/site-packages/anvil_downlink_worker/anvil/_server.py:1441

2022-08-21 19:50:54,972 [ERROR anvil.app-server.run] Error report from server code:
FileNotFoundError: [Errno 2] No such file or directory: 'leagues/league4/team-lineups/LAQ.json'
Traceback:
  MyCRUDApp/ServerModule1.py:18

  c:/users/markd/appdata/local/programs/python/python38/lib/site-packages/anvil_downlink_worker/anvil/_server.py:1441

2022-08-21 19:51:15,130 [ERROR anvil.app-server.run] Error report from server code:
FileNotFoundError: [Errno 2] No such file or directory: 'leagues/league4/team-lineups/liverpool-ale-quaffers.roster'
Traceback:
  MyCRUDApp/ServerModule1.py:23

  c:/users/markd/appdata/local/programs/python/python38/lib/site-packages/anvil_downlink_worker/anvil/_server.py:1441

2022-08-21 19:52:55,175 [ERROR anvil.app-server.run] Error report from server code:
ModuleNotFoundError: No module named 'jsbeautifier'
Traceback:
  MyCRUDApp/ServerModule1.py:13

  c:/users/markd/appdata/local/programs/python/python38/lib/site-packages/anvil_downlink_worker/__init__.py:70

  c:/users/markd/appdata/local/programs/python/python38/lib/site-packages/anvil_downlink_worker/__init__.py:181

2022-08-22 00:13:03,925 [ERROR anvil.app-server.run] Error report from client code:
AttributeError: 'Form1' object has no attribute 'content_panel'
Traceback:
  app/MyCRUDApp/Form1/_template.py:49

  app/MyCRUDApp/Form1/__init__.py:10

2022-08-22 00:13:49,330 [ERROR anvil.app-server.run] Error report from client code:
AttributeError: 'Form1' object has no attribute 'tasks_grid'
Traceback:
  app/MyCRUDApp/Form1/_template.py:33

  app/MyCRUDApp/Form1/__init__.py:10

2022-08-22 00:14:10,710 [ERROR anvil.app-server.run] Error report from client code:
AttributeError: 'Form1' object has no attribute 'tasks_grid'
Traceback:
  app/MyCRUDApp/Form1/_template.py:42

  app/MyCRUDApp/Form1/__init__.py:10

2022-08-22 00:14:52,285 [ERROR anvil.app-server.run] Error report from client code:
AttributeError: 'Form1' object has no attribute 'card_1'
Traceback:
  app/MyCRUDApp/Form1/_template.py:43

  app/MyCRUDApp/Form1/__init__.py:10

2022-08-22 00:17:17,499 [ERROR anvil.app-server.run] Error report from client code:
AttributeError: 'Form1' object has no attribute 'tasks_panel'
Traceback:
  app/MyCRUDApp/Form1/_template.py:33

  app/MyCRUDApp/Form1/__init__.py:10

2022-08-22 00:23:16,123 [ERROR anvil.app-server.run] Error report from client code:
ValueError: Cannot set event handler for for unknown event 'pressed_enter' on TextArea component. Custom event names must start with 'x-'.
Traceback:
  app/MyCRUDApp/Form1/__init__.py:16

2022-08-22 00:24:14,872 [ERROR anvil.app-server.run] Error report from client code:
NoServerFunctionError: No server function matching "add_task" has been registered
Traceback:
  app/MyCRUDApp/Form1/__init__.py:28

2022-08-22 00:25:19,441 [ERROR anvil.app-server.run] Error report from client code:
TypeError: get_lineup_click() got an unexpected keyword argument 'keys'
Traceback:
2022-08-22 00:30:25,934 [ERROR anvil.app-server.run] Error report from client code:
AttributeError: 'Form1' object has no attribute 'set_lineup_panel'
Traceback:
  app/MyCRUDApp/Form1/_template.py:27

  app/MyCRUDApp/Form1/__init__.py:10

2022-08-22 00:33:21,394 [ERROR anvil.app-server.run] Error report from client code:
AnvilWrappedError: string indices must be integers
Traceback:
  MyCRUDApp/ServerModule1.py:58

  app/MyCRUDApp/Form1/__init__.py:28

2022-08-22 00:35:04,695 [ERROR anvil.app-server.run] Error report from client code:
AnvilWrappedError: string indices must be integers
Traceback:
  MyCRUDApp/ServerModule1.py:58

  app/MyCRUDApp/Form1/__init__.py:28

2022-08-22 00:35:22,672 [ERROR anvil.app-server.run] Error report from client code:
TypeError: 'Button' object is not callable
Traceback:
  app/MyCRUDApp/Form1/__init__.py:30

2022-08-22 00:37:04,045 [ERROR anvil.app-server.run] Error report from client code:
TypeError: 'Button' object is not callable
Traceback:
  app/MyCRUDApp/Form1/__init__.py:30

2022-08-22 00:45:07,302 [ERROR anvil.app-server.run] Error report from client code:
ModuleNotFoundError: No module named 'ItemTemplate1'
Traceback:
  app/MyCRUDApp/Form1/_template.py:2

  app/MyCRUDApp/Form1/__init__.py:1

2022-08-22 00:47:29,859 [ERROR anvil.app-server.run] Error report from client code:
AttributeError: 'Form1' object has no attribute 'spacer'
Traceback:
  app/MyCRUDApp/Form1/_template.py:34

  app/MyCRUDApp/Form1/__init__.py:9

2022-08-22 00:59:19,772 [ERROR anvil.app-server.run] Error report from client code:
ModuleNotFoundError: No module named 'Form1'
Traceback:
2022-08-22 01:00:11,609 [ERROR anvil.app-server.run] Error report from client code:
AttributeError: "LineupChangeForm" module does not contain a class called "LineupChangeForm"
Traceback:
2022-08-22 01:01:03,511 [ERROR anvil.app-server.run] Error report from client code:
AttributeError: 'LineupChangeForm' object has no attribute 'card_2'
Traceback:
  app/MyCRUDApp/LineupChangeForm/_template.py:36

  app/MyCRUDApp/LineupChangeForm/__init__.py:8

2022-08-24 23:54:01,776 [ERROR anvil.app-server.run] Error report from client code:
TypeError: ColumnPanel got unexpected keyword argument(s): 'align'
Traceback:
  app/FantasyLineupSetter/LineupChangeForm/_template.py:15

  app/FantasyLineupSetter/LineupChangeForm/_template.py:21

  app/FantasyLineupSetter/LineupChangeForm/__init__.py:8

2022-08-24 23:55:10,963 [ERROR anvil.app-server.run] Error report from client code:
AttributeError: 'LineupChangeForm' object has no attribute 'link_lineup'
Traceback:
  app/FantasyLineupSetter/LineupChangeForm/__init__.py:13

2022-08-24 23:59:51,960 [ERROR anvil.app-server.run] Error report from client code:
ValueError: This component is already added to a container, call remove_from_parent() first
Traceback:
  app/FantasyLineupSetter/LineupChangeForm/_template.py:23

  app/FantasyLineupSetter/LineupChangeForm/_template.py:28

  app/FantasyLineupSetter/LineupChangeForm/__init__.py:8

2022-08-25 00:04:00,018 [ERROR anvil.app-server.run] Error report from client code:
AttributeError: 'LineupChangeForm' object has no attribute 'main_content'
Traceback:
  app/FantasyLineupSetter/LineupChangeForm/_template.py:31

  app/FantasyLineupSetter/LineupChangeForm/__init__.py:8

2022-08-25 00:07:53,472 [ERROR anvil.app-server.run] Error report from client code:
AttributeError: 'LineupChangeForm' object has no attribute 'link_drop'
Traceback:
  app/FantasyLineupSetter/LineupChangeForm/__init__.py:15

2022-08-25 00:18:56,526 [ERROR anvil.app-server.run] Error report from client code:
AttributeError: 'LineupChangeForm' object has no attribute 'get_roster'
Traceback:
  app/FantasyLineupSetter/LineupChangeForm/__init__.py:13

2022-08-25 00:20:10,043 [ERROR anvil.app-server.run] Error report from client code:
AttributeError: 'LineupChangeForm' object has no attribute 'add_lineup'
Traceback:
  app/FantasyLineupSetter/LineupChangeForm/__init__.py:14

2022-08-25 00:21:07,834 [ERROR anvil.app-server.run] Error report from client code:
ValueError: This component is already added to a container, call remove_from_parent() first
Traceback:
  app/FantasyLineupSetter/LineupChangeForm/_template.py:51

2022-08-25 00:21:46,744 [ERROR anvil.app-server.run] Error report from client code:
ValueError: This component is already added to a container, call remove_from_parent() first
Traceback:
  app/FantasyLineupSetter/LineupChangeForm/_template.py:63

2022-08-25 00:26:48,452 [ERROR anvil.app-server.run] Error report from client code:
AttributeError: 'LineupChangeForm' object has no attribute 'set_lineup'
Traceback:
  app/FantasyLineupSetter/LineupChangeForm/__init__.py:9

2022-08-25 00:28:44,430 [ERROR anvil.app-server.run] Error report from client code:
TypeError: 'LineupChangeForm' object is not iterable
Traceback:
  app/FantasyLineupSetter/LineupChangeForm/_template.py:94

  app/FantasyLineupSetter/LineupChangeForm/__init__.py:9

2022-08-25 00:29:27,043 [ERROR anvil.app-server.run] Error report from client code:
AttributeError: 'LineupChangeForm' object has no attribute 'get_lineup'
Traceback:
  app/FantasyLineupSetter/LineupChangeForm/_template.py:96

  app/FantasyLineupSetter/LineupChangeForm/__init__.py:9

2022-08-25 00:30:47,445 [ERROR anvil.app-server.run] Error report from client code:
SyntaxError: bad input
Traceback:
  app/FantasyLineupSetter/LineupChangeForm/_template.py:97

  app/FantasyLineupSetter/LineupChangeForm/__init__.py:1

2022-08-25 00:32:27,240 [ERROR anvil.app-server.run] Error report from client code:
AnvilWrappedError: not readable
Traceback:
  FantasyLineupSetter/ServerModule1.py:103

  app/FantasyLineupSetter/LineupChangeForm/__init__.py:25

2022-08-25 00:33:41,699 [ERROR anvil.app-server.run] Error report from client code:
AnvilWrappedError: not readable
Traceback:
  FantasyLineupSetter/ServerModule1.py:103

  app/FantasyLineupSetter/LineupChangeForm/__init__.py:25

2022-08-25 00:38:51,466 [ERROR anvil.app-server.run] Error report from client code:
AnvilWrappedError: write() argument must be str, not TextBox
Traceback:
  FantasyLineupSetter/ServerModule1.py:109

  app/FantasyLineupSetter/LineupChangeForm/__init__.py:25

2022-08-25 00:41:20,670 [ERROR anvil.app-server.run] Error report from client code:
AnvilWrappedError: write() argument must be str, not TextBox
Traceback:
  FantasyLineupSetter/ServerModule1.py:110

  app/FantasyLineupSetter/LineupChangeForm/__init__.py:25

2022-08-25 00:59:09,502 [ERROR anvil.app-server.run] Error report from client code:
AttributeError: 'LineupChangeForm' object has no attribute 'init_components_lineup'
Traceback:
  app/FantasyLineupSetter/LineupChangeForm/_template.py:112

  app/FantasyLineupSetter/LineupChangeForm/__init__.py:10

2022-08-25 00:59:57,495 [ERROR anvil.app-server.run] Error report from client code:
ValueError: This component is already added to a container, call remove_from_parent() first
Traceback:
  app/FantasyLineupSetter/LineupChangeForm/_template.py:76

2022-08-25 01:00:02,339 [ERROR anvil.app-server.run] Error report from client code:
ValueError: This component is already added to a container, call remove_from_parent() first
Traceback:
  app/FantasyLineupSetter/LineupChangeForm/_template.py:46

  app/FantasyLineupSetter/LineupChangeForm/_template.py:50

2022-08-25 01:02:33,342 [ERROR anvil.app-server.run] Error report from client code:
ValueError: This component is already added to a container, call remove_from_parent() first
Traceback:
  app/FantasyLineupSetter/LineupChangeForm/_template.py:46

  app/FantasyLineupSetter/LineupChangeForm/_template.py:62

2022-08-25 01:02:59,264 [ERROR anvil.app-server.run] Error report from client code:
ValueError: This component is already added to a container, call remove_from_parent() first
Traceback:
  app/FantasyLineupSetter/LineupChangeForm/_template.py:46

  app/FantasyLineupSetter/LineupChangeForm/_template.py:62

2022-08-25 01:03:20,497 [ERROR anvil.app-server.run] Error report from client code:
ValueError: This component is already added to a container, call remove_from_parent() first
Traceback:
  app/FantasyLineupSetter/LineupChangeForm/_template.py:68

2022-08-25 01:04:28,239 [ERROR anvil.app-server.run] Error report from client code:
AttributeError: 'LineupChangeForm' object has no attribute 'self'
Traceback:
  app/FantasyLineupSetter/LineupChangeForm/_template.py:55

  app/FantasyLineupSetter/LineupChangeForm/__init__.py:9

2022-08-25 01:04:47,356 [ERROR anvil.app-server.run] Error report from client code:
ExternalError: HierarchyRequestError: Failed to execute 'appendChild' on 'Node': The new child element contains the parent.
Traceback:
  app/FantasyLineupSetter/LineupChangeForm/_template.py:46

  app/FantasyLineupSetter/LineupChangeForm/_template.py:55

  app/FantasyLineupSetter/LineupChangeForm/__init__.py:9

2022-08-25 11:30:17,831 [ERROR anvil.app-server.run] Error report from client code:
TypeError: FlowPanel got unexpected keyword argument(s): 'slot'
Traceback:
  app/FantasyLineupSetter/LineupChangeForm/_template.py:38

  app/FantasyLineupSetter/LineupChangeForm/__init__.py:8

2022-08-25 11:35:26,336 [ERROR anvil.app-server.run] Error report from client code:
TypeError: FlowPanel got unexpected keyword argument(s): 'slot'
Traceback:
  app/FantasyLineupSetter/LineupChangeForm/_template.py:15

  app/FantasyLineupSetter/LineupChangeForm/__init__.py:8

2022-08-25 11:40:06,739 [ERROR anvil.app-server.run] Error report from client code:
AttributeError: 'TextBox' object has no attribute 'show'
Traceback:
  app/FantasyLineupSetter/LineupChangeForm/_template.py:41

  app/FantasyLineupSetter/LineupChangeForm/__init__.py:8

2022-08-25 11:42:03,778 [ERROR anvil.app-server.run] Error report from client code:
AttributeError: 'TextBox' object has no attribute 'show'
Traceback:
  app/FantasyLineupSetter/LineupChangeForm/_template.py:41

  app/FantasyLineupSetter/LineupChangeForm/__init__.py:8

2022-08-25 18:25:10,728 [ERROR anvil.app-server.run] Error report from client code:
TypeError: set_event_handler() takes exactly 2 arguments (3 given)
Traceback:
  app/FantasyLineupSetter/LineupChangeForm/_template.py:102

  app/FantasyLineupSetter/LineupChangeForm/__init__.py:10

2022-08-25 18:27:25,800 [ERROR anvil.app-server.run] Error report from client code:
TypeError: <lambda>() got an unexpected keyword argument 'keys'
Traceback:
2022-08-25 18:27:26,426 [ERROR anvil.app-server.run] Error report from client code:
TypeError: <lambda>() got an unexpected keyword argument 'keys'
Traceback:
2022-08-26 00:17:44,302 [ERROR anvil.app-server.run] Error report from client code:
TypeError: The 'click' event handler added to Button must be a callable, not type 'Button'
Traceback:
  app/FantasyLineupSetter/LineupChangeForm/_template.py:121

  app/FantasyLineupSetter/LineupChangeForm/__init__.py:10

2022-08-26 00:18:07,938 [ERROR anvil.app-server.run] Error report from client code:
AttributeError: 'DropDown' object has no attribute 'text'
Traceback:
  app/FantasyLineupSetter/LineupChangeForm/__init__.py:33

2022-08-26 00:18:40,171 [ERROR anvil.app-server.run] Error report from client code:
AttributeError: 'DropDown' object has no attribute 'value'
Traceback:
  app/FantasyLineupSetter/LineupChangeForm/__init__.py:34

2022-08-26 00:20:12,436 [ERROR anvil.app-server.run] Error report from client code:
AnvilWrappedError: local variable 'ret' referenced before assignment
Traceback:
  FantasyLineupSetter/ServerModule1.py:148

  app/FantasyLineupSetter/LineupChangeForm/__init__.py:34

2022-08-26 00:21:04,290 [ERROR anvil.app-server.run] Error report from client code:
AnvilWrappedError: local variable 'ret' referenced before assignment
Traceback:
  FantasyLineupSetter/ServerModule1.py:149

  app/FantasyLineupSetter/LineupChangeForm/__init__.py:33

2022-08-26 00:23:34,344 [ERROR anvil.app-server.run] Error report from client code:
AnvilWrappedError: local variable 'ret' referenced before assignment
Traceback:
  FantasyLineupSetter/ServerModule1.py:149

  app/FantasyLineupSetter/LineupChangeForm/__init__.py:33

2022-08-26 00:24:41,418 [ERROR anvil.app-server.run] Error report from client code:
AnvilWrappedError: [Errno 2] No such file or directory: 'leagues/livetestleague/debug_output/NYB_wk0_totals.json'
Traceback:
  FantasyLineupSetter/ServerModule1.py:125

  app/FantasyLineupSetter/LineupChangeForm/__init__.py:33

2022-08-26 00:36:25,843 [ERROR anvil.app-server.run] Error report from client code:
AnvilWrappedError: [Errno 2] No such file or directory: 'leagues/livetestleague/team-lineups/next_team.json'
Traceback:
  FantasyLineupSetter/ServerModule1.py:69

  app/FantasyLineupSetter/LineupChangeForm/__init__.py:23

2022-08-26 00:36:28,622 [ERROR anvil.app-server.run] Error report from client code:
AnvilWrappedError: [Errno 2] No such file or directory: 'leagues/livetestleague/team-lineups/next_team.json'
Traceback:
  FantasyLineupSetter/ServerModule1.py:22

  app/FantasyLineupSetter/LineupChangeForm/__init__.py:29

2022-08-26 00:37:06,784 [ERROR anvil.app-server.run] Error report from client code:
AnvilWrappedError: [Errno 2] No such file or directory: 'leagues/livetestleague/team-lineups/next_team.json'
Traceback:
  FantasyLineupSetter/ServerModule1.py:22

  app/FantasyLineupSetter/LineupChangeForm/__init__.py:29

2022-08-26 00:37:12,461 [ERROR anvil.app-server.run] Error report from client code:
AnvilWrappedError: [Errno 2] No such file or directory: 'leagues/livetestleague/team-lineups/next_team.json'
Traceback:
  FantasyLineupSetter/ServerModule1.py:36

  app/FantasyLineupSetter/LineupChangeForm/__init__.py:14

2022-08-26 01:29:22,352 [ERROR anvil.app-server.run] Error report from client code:
AnvilWrappedError: 'charmap' codec can't decode byte 0x81 in position 9395: character maps to <undefined>
Traceback:
  c:/users/markd/appdata/local/programs/python/python38/lib/encodings/cp1252.py:23

  FantasyLineupSetter/ServerModule1.py:142

  app/FantasyLineupSetter/LineupChangeForm/__init__.py:33

2022-08-26 01:35:12,545 [ERROR anvil.app-server.run] Error report from client code:
AnvilWrappedError: [Errno 2] No such file or directory: 'leagues/livetestleague/team-lineups/next_team.json'
Traceback:
  FantasyLineupSetter/ServerModule1.py:22

  app/FantasyLineupSetter/LineupChangeForm/__init__.py:29

2022-08-26 01:35:58,893 [ERROR anvil.app-server.run] Error report from client code:
AnvilWrappedError: [Errno 2] No such file or directory: 'leagues/livetestleague/debug_output/_wk_totals.json'
Traceback:
  FantasyLineupSetter/ServerModule1.py:125

  app/FantasyLineupSetter/LineupChangeForm/__init__.py:33

2022-08-26 01:42:16,357 [ERROR anvil.app-server.run] Error report from client code:
AttributeError: 'LineupChangeForm' object has no attribute 'chat_box'
Traceback:
  app/FantasyLineupSetter/LineupChangeForm/_template.py:76

  app/FantasyLineupSetter/LineupChangeForm/__init__.py:8

2022-08-26 01:45:03,513 [ERROR anvil.app-server.run] Error report from client code:
AttributeError: 'LineupChangeForm' object has no attribute 'chat_msg'
Traceback:
  app/FantasyLineupSetter/LineupChangeForm/_template.py:79

  app/FantasyLineupSetter/LineupChangeForm/__init__.py:8

2022-08-26 01:57:40,807 [ERROR anvil.app-server.run] Error report from client code:
AnvilWrappedError: [Errno 2] No such file or directory: 'leagues/livetestleague/debug_output/League_note'
Traceback:
  FantasyLineupSetter/ServerModule1.py:136

  app/FantasyLineupSetter/LineupChangeForm/__init__.py:33

2022-08-26 12:05:48,214 [ERROR anvil.app-server.run] Error report from client code:
AttributeError: 'FlowPanel' object has no attribute 'add'
Traceback:
  app/FantasyLineupSetter/LineupChangeForm/_template.py:74

  app/FantasyLineupSetter/LineupChangeForm/__init__.py:8

2022-08-26 12:15:24,155 [ERROR anvil.app-server.run] Error report from client code:
AttributeError: 'FlowPanel' object has no attribute 'children'
Traceback:
  app/FantasyLineupSetter/LineupChangeForm/_template.py:84

  app/FantasyLineupSetter/LineupChangeForm/__init__.py:8

2022-08-26 16:53:34,505 [ERROR anvil.app-server.run] Error report from client code:
TypeError: string indices must be integers, not str
Traceback:
  app/FantasyLineupSetter/LineupChangeForm/__init__.py:16

2022-08-26 16:54:02,894 [ERROR anvil.app-server.run] Error report from client code:
TypeError: string indices must be integers, not str
Traceback:
  app/FantasyLineupSetter/LineupChangeForm/__init__.py:17

2022-08-26 17:00:11,888 [ERROR anvil.app-server.run] Error report from client code:
TypeError: cannot concatenate 'str' and 'int' objects
Traceback:
  app/FantasyLineupSetter/LineupChangeForm/__init__.py:35

2022-08-26 17:00:34,593 [ERROR anvil.app-server.run] Error report from client code:
TypeError: cannot concatenate 'str' and 'int' objects
Traceback:
  app/FantasyLineupSetter/LineupChangeForm/__init__.py:35

2022-08-26 17:04:22,507 [ERROR anvil.app-server.run] Error report from client code:
IndexError: pop index out of range
Traceback:
  app/FantasyLineupSetter/LineupChangeForm/__init__.py:25

2022-08-26 17:05:16,578 [ERROR anvil.app-server.run] Error report from client code:
IndexError: pop index out of range
Traceback:
  app/FantasyLineupSetter/LineupChangeForm/__init__.py:25

2022-08-26 17:23:36,099 [ERROR anvil.app-server.run] Error report from client code:
AttributeError: 'TextBox' object has no attribute 'replace'
Traceback:
  app/FantasyLineupSetter/LineupChangeForm/__init__.py:58

2022-08-26 17:24:26,594 [ERROR anvil.app-server.run] Error report from client code:
AttributeError: 'str' object has no attribute 'trim'
Traceback:
  app/FantasyLineupSetter/LineupChangeForm/__init__.py:62

2022-08-26 17:25:13,060 [ERROR anvil.app-server.run] Error report from client code:
AnvilWrappedError: list index out of range
Traceback:
  C:/Users/markd/PycharmProjects/fantasy-baseball-scramble/mlb_api.py:63

  FantasyLineupSetter/ServerModule1.py:63

  app/FantasyLineupSetter/LineupChangeForm/__init__.py:67

2022-08-26 17:26:01,398 [ERROR anvil.app-server.run] Error report from client code:
AnvilWrappedError: list index out of range
Traceback:
  C:/Users/markd/PycharmProjects/fantasy-baseball-scramble/mlb_api.py:63

  FantasyLineupSetter/ServerModule1.py:63

  app/FantasyLineupSetter/LineupChangeForm/__init__.py:68

2022-08-26 17:27:26,437 [ERROR anvil.app-server.run] Error report from client code:
AnvilWrappedError: list index out of range
Traceback:
  C:/Users/markd/PycharmProjects/fantasy-baseball-scramble/mlb_api.py:64

  FantasyLineupSetter/ServerModule1.py:63

  app/FantasyLineupSetter/LineupChangeForm/__init__.py:68

2022-08-26 17:29:43,437 [ERROR anvil.app-server.run] Error report from client code:
AnvilWrappedError: list index out of range
Traceback:
  C:/Users/markd/PycharmProjects/fantasy-baseball-scramble/mlb_api.py:63

  FantasyLineupSetter/ServerModule1.py:63

  app/FantasyLineupSetter/LineupChangeForm/__init__.py:69

2022-08-26 18:43:52,026 [ERROR anvil.app-server.run] Error report from client code:
AnvilWrappedError: can only concatenate str (not "NoneType") to str
Traceback:
  FantasyLineupSetter/ServerModule1.py:30

  app/FantasyLineupSetter/LineupChangeForm/__init__.py:85

2022-08-26 20:20:24,423 [ERROR anvil.app-server.run] Error report from client code:
AnvilWrappedError: can only concatenate str (not "tuple") to str
Traceback:
  FantasyLineupSetter/ServerModule1.py:31

  app/FantasyLineupSetter/LineupChangeForm/__init__.py:85

2022-08-26 20:20:26,495 [ERROR anvil.app-server.run] Error report from client code:
AnvilWrappedError: can only concatenate str (not "tuple") to str
Traceback:
  FantasyLineupSetter/ServerModule1.py:31

  app/FantasyLineupSetter/LineupChangeForm/__init__.py:85

2022-08-26 21:21:26,100 [ERROR anvil.app-server.run] Error report from client code:
AnvilWrappedError: Marcus Stroman is in the lineup but not on their roster.
Traceback:
  C:/Users/markd/PycharmProjects/fantasy-baseball-scramble/mlb_api.py:12

  C:/Users/markd/PycharmProjects/fantasy-baseball-scramble/mlb_api.py:64

  FantasyLineupSetter/ServerModule1.py:43

  app/FantasyLineupSetter/LineupChangeForm/__init__.py:85

2022-08-26 21:23:46,978 [ERROR anvil.app-server.run] Error report from client code:
AnvilWrappedError: Marcus Stroman is in the lineup but not on their roster.
Traceback:
  C:/Users/markd/PycharmProjects/fantasy-baseball-scramble/mlb_api.py:12

  C:/Users/markd/PycharmProjects/fantasy-baseball-scramble/mlb_api.py:64

  FantasyLineupSetter/ServerModule1.py:43

  app/FantasyLineupSetter/LineupChangeForm/__init__.py:85

2022-08-26 21:24:25,944 [ERROR anvil.app-server.run] Error report from client code:
AnvilWrappedError: Marcus Stroman is in the lineup but not on their roster.
Traceback:
  C:/Users/markd/PycharmProjects/fantasy-baseball-scramble/mlb_api.py:12

  C:/Users/markd/PycharmProjects/fantasy-baseball-scramble/mlb_api.py:65

  FantasyLineupSetter/ServerModule1.py:43

  app/FantasyLineupSetter/LineupChangeForm/__init__.py:85

2022-08-26 21:25:54,796 [ERROR anvil.app-server.run] Error report from client code:
AnvilWrappedError: Josh Hader is in the lineup but not on their roster.
Traceback:
  C:/Users/markd/PycharmProjects/fantasy-baseball-scramble/mlb_api.py:12

  C:/Users/markd/PycharmProjects/fantasy-baseball-scramble/mlb_api.py:65

  FantasyLineupSetter/ServerModule1.py:45

  app/FantasyLineupSetter/LineupChangeForm/__init__.py:85

2022-08-26 21:27:16,830 [ERROR anvil.app-server.run] Error report from client code:
AnvilWrappedError: list.remove(x): x not in list
Traceback:
  FantasyLineupSetter/ServerModule1.py:99

  app/FantasyLineupSetter/LineupChangeForm/__init__.py:77

2022-08-26 21:29:27,103 [ERROR anvil.app-server.run] Error report from client code:
AnvilWrappedError: Shohei Ohtani is in the lineup but not on their roster.
Traceback:
  C:/Users/markd/PycharmProjects/fantasy-baseball-scramble/mlb_api.py:12

  C:/Users/markd/PycharmProjects/fantasy-baseball-scramble/mlb_api.py:65

  FantasyLineupSetter/ServerModule1.py:53

  app/FantasyLineupSetter/LineupChangeForm/__init__.py:85

2022-08-26 21:29:42,063 [ERROR anvil.app-server.run] Error report from client code:
AnvilWrappedError: Shohei Ohtani is in the lineup but not on their roster.
Traceback:
  C:/Users/markd/PycharmProjects/fantasy-baseball-scramble/mlb_api.py:12

  C:/Users/markd/PycharmProjects/fantasy-baseball-scramble/mlb_api.py:65

  FantasyLineupSetter/ServerModule1.py:53

  app/FantasyLineupSetter/LineupChangeForm/__init__.py:85

